runit/fluentd.sh

```diff
@@ -1,3 +1,5 @@
 #!/bin/sh
 
-exec /usr/local/bin/fluentd --use-v1-config -c /etc/fluent/fluent.conf -vv >>/var/log/fluentd.log 2>&1
+if [[ -n "$KINESIS_STREAM" && -n "$AWS_ACCESS_KEY_ID" && -n "$AWS_SECRET_ACCESS_KEY" && -n "$KINESIS_REGION"]] ; then
```

Comment:
I also think the syntax should be this instead:
`if [[ -n "$KINESIS_STREAM" && -n "$AWS_ACCESS_KEY_ID" && -n "$AWS_SECRET_ACCESS_KEY" && -n "$KINESIS_REGION" ]]; then
`